website/settings/defaults.py

```diff
@@ -129,7 +129,11 @@ def parent_dir(path):
 
 USE_EMAIL = True
 FROM_EMAIL = 'openscienceframework-noreply@osf.io'
-SUPPORT_EMAIL = 'support@osf.io'
+
+# support email
+OSF_SUPPORT_EMAIL = 'support@osf.io'
+COS_SUPPORT_EMAIL = 'support@cos.io'
```

Comment:
So I wasn't 100% clear in the ticket, but the intention was for support@cos.io and support@osf.io to be replaced with the single variable. support@cos.io goes to the same place as support@osf.io and there is not a consistent pattern to where they occur so support@cos.io can be removed.